### ISSUE TITLE
[request]: Pre-sign DeleteObject URLs

### ISSUE THREAD

**Issue by stefanhoelzl** [awslabs/aws-sdk-rust#473]
### Community Note
* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue, please leave a comment

### Tell us about your request

I want to be able to pre-signed a URL for a DeleteObject command.

### Tell us about the problem you're trying to solve.

Allowing users without access keys to delete certain objects from my buckets with pre-signed URLs

### Are you currently working around this issue?

Delete operations are routed through my backend.

**Comment by rcoh** [comment]
we're working on it thanks for reporting!

**Comment by rcoh** [comment]
fix was merged, you can now use this in the next branch:
```
[dependencies]
aws-config = { git = "https://github.com/awslabs/aws-sdk-rust", branch = "next" }
aws-sdk-s3 = { git = "https://github.com/awslabs/aws-sdk-rust", branch = "next" }
```

**Comment by jdisanti** [comment]
This was released with [`v0.9.0`](https://github.com/awslabs/aws-sdk-rust/releases/tag/v0.9.0).
